biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceBuilder.java

```diff
@@ -509,22 +513,44 @@ public Capability addProvideCapability(String namespace, Attrs attrs) throws Exc
 		return addCapability0(capb);
 	}
 
+	/**
+	 * Adds a set of package exports assuming they are from the system bundle.
+     *
+	 * @deprecated Use {@link #addExportPackages(Parameters, String, String)} in preference.
+	 */
+	@Deprecated
```

Comment:
Why add a new method that is deprecated. That does not make sense.

Reply:
The intention was to create a method that can be used in contexts where the bundle identity is not known, to avoid a more significant refactoring.